kv/kvserver/loqrecovery: TestRetrieveApplyStatus failed

kv/kvserver/loqrecovery.TestRetrieveApplyStatus [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9065784?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9065784?buildTab=artifacts#/) on master @ [7f2afbcb1b7aef600fa3506649b9d3383a2f5bf9](https://github.com/cockroachdb/cockroach/commits/7f2afbcb1b7aef600fa3506649b9d3383a2f5bf9):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 245772 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRetrieveApplyStatus
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/ea0927e04eade2740899effbae326bb2/logTestRetrieveApplyStatus4175440691
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/replication
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRetrieveApplyStatus.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-25371

--- a/pkg/kv/kvserver/loqrecovery/server_integration_test.go
+++ b/pkg/kv/kvserver/loqrecovery/server_integration_test.go
@@ -568,7 +568,8 @@ func TestRetrieveApplyStatus(t *testing.T) {
 
 	// Use scratch range to ensure we have a range that loses quorum.
 	sk := tc.ScratchRange(t)
-	require.NoError(t, tc.WaitForFullReplication(), "failed to wait for full replication")
+	require.NoError(t, tc.WaitFor5NodeReplication(),
+		"failed to wait for full replication of 5 node cluster")
 	tc.ToggleReplicateQueues(false)
 	d := tc.LookupRangeOrFatal(t, sk)
 

